vtex-apps/lightspeed
# [LIGHTSPD-40] Fixed nightly sync

**What problem is this solving?**

The nightly sync on Redoxx got errors `unhandledRejection TypeError: Cannot read property 'customSku' of null` due to some null items in the process queue. 

**How should this be manually tested?**
1. run [https://aurora--redoxx.myvtex.com/lightspeed/_v/0/process-sync](url)
2. go to the Splunk for 'index=io_vtex_logs app="vtex.lightspeed@0.8.1" account=redoxx workspace=aurora' you shouldn't see error levels for the process.
